bundles/org.openhab.binding.gardena/src/main/java/org/openhab/binding/gardena/internal/handler/GardenaAccountHandler.java

```diff
@@ -51,80 +66,189 @@
+        } catch (DateTimeParseException e) {
+            apiCallSuppressionUntil = null;
+        }
+    }
+
+    /**
+     * Get the duration remaining until the end of the api call suppression window, or Duration.ZERO if we are outside
+     * the call suppression window.
+     *
+     * @return the duration until the end of the suppression window, or zero.
+     */
+    private Duration apiCallSuppressionDelay() {
+        Instant now = Instant.now();
+        Instant until = apiCallSuppressionUntil;
+        return (until != null) && now.isBefore(until) ? Duration.between(now, until) : Duration.ZERO;
+    }
+
+    /**
+     * Updates the time when api call suppression ends to now() plus the given delay. If delay is zero or negative, the
+     * suppression time is nulled. Saves the value as a property to ensure consistent behaviour across restarts.
+     *
+     * @param delay the delay until the end of the suppression window.
+     */
+    private void apiCallSuppressionUpdate(Duration delay) {
+        Instant until = (delay.isZero() || delay.isNegative()) ? null : Instant.now().plus(delay);
+        getThing().setProperty(GardenaBindingConstants.API_CALL_SUPPRESSION_UNTIL,
+                until == null ? null : until.toString());
+        apiCallSuppressionUntil = until;
     }
 
     @Override
     public void initialize() {
         logger.debug("Initializing Gardena account '{}'", getThing().getUID().getId());
-        initializeGardena();
+        loadApiCallSuppressionUntil();
+        Duration delay = apiCallSuppressionDelay();
+        if (delay.isZero()) {
+            // do immediate initialisation
+            scheduler.submit(() -> initializeGardena());
+        } else {
+            // delay the initialisation
+            scheduleReinitialize(delay);
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, getUiText());
+        }
     }
 
     public void setDiscoveryService(GardenaDeviceDiscoveryService discoveryService) {
         this.discoveryService = discoveryService;
     }
 
+    /**
+     * Format a localized explanatory description regarding active call suppression.
+     *
+     * @return the localized description text, or null if call suppression is not active.
+     */
+    private @Nullable String getUiText() {
+        Instant until = apiCallSuppressionUntil;
+        if (until != null) {
+            ZoneId zone = timeZoneProvider.getTimeZone();
+            boolean isToday = LocalDate.now(zone).equals(LocalDate.ofInstant(until, zone));
+            DateTimeFormatter formatter = isToday ? DateTimeFormatter.ofLocalizedTime(FormatStyle.MEDIUM)
+                    : DateTimeFormatter.ofLocalizedDateTime(FormatStyle.MEDIUM);
+            String value = i18nProvider.getText(bundle, RECONNECT_MSG_KEY, null, localeProvider.getLocale(),
+                    formatter.format(ZonedDateTime.ofInstant(until, zone)));
+            return value;
```

Comment:
This should work:
```suggestion
            return "@text/accounthandler.waiting-until-to-reconnect [\"" + formatter.format(ZonedDateTime.ofInstant(until, zone)) + "\"]";
                    );
```

Reply:
So I don't need to capture the localProvider or the i18nProvider in the thing factory component, nor propagate them into the created Things? Or ??

Reply:
There would be no more usages of them, so correct. If you would like to eliminate them, please verify my suggestion. The magic happens within `updateStatus()`, so we just need to provide a properly formatted string for it.

Reply:
It may actually be quite difficult to test your localization magic code because you would have to have a Gardena account, and it would have to be in lockout mode. ... ...

Reply:
It the variable substitution works correctly in English, I have no doubts that it will also work for other languages. This happens completely outside your code now, so as an additional advantage, there is less to test. :)

Reply:
PS I grepped the addons folder for the following Regex and found about a dozen classes where your syntax is used. So even if we cannot test it completely, we are in good company using this code form. :)
```
(?s)"@text/.*\[\\"".*"\\"\]
```